setup.py

```diff
@@ -81,6 +81,7 @@ def read(path, encoding="utf-8"):
     "protobuf",
     "bs4",
     "openai>=0.27.4",  # CHATGPT
```

Comment:
Does `litellm` subsumes the `openai`? Or we still need both dependency?

Reply:
litellm has openai as a dependency